src/server/components/formRunner/helpers.ts

```diff
@@ -2,26 +2,32 @@ import * as FormRunner from "./types";
 import path from "path";
 import fs from "fs";
 import {
-  LawyerListItemGetObject,
   FuneralDirectorListItemGetObject,
-  TranslatorInterpreterListItemGetObject,
+  LawyerListItemGetObject,
   List,
-  BaseListItemGetObject,
+  ListItem,
   ServiceType,
+  TranslatorInterpreterListItemGetObject,
 } from "server/models/types";
 import * as lawyers from "./lawyers";
 import * as funeralDirectors from "./funeralDirectors";
 import * as translatorsInterpreters from "./translatorsInterpreters";
 import { kebabCase } from "lodash";
+import { isLocalHost, SERVICE_DOMAIN } from "server/config";
+import { createFormRunnerEditListItemLink, createFormRunnerReturningUserLink } from "server/components/lists/helpers";
+import { getInitiateFormRunnerSessionToken } from "server/components/dashboard/helpers";
 
 export function getNewSessionWebhookData(
   listType: string,
   listItemId: number,
   questions: Array<Partial<FormRunner.Question>> | undefined,
-  message: string
+  message: string,
+  isAnnualReview: boolean | undefined,
+  listItemRef: string
 ): FormRunner.NewSessionData {
-  const callbackUrl = `http://lists:3000/ingest/${listType}/${listItemId}`;
-  const redirectPath = `/summary`;
+  const callbackUrl = `http://localhost:3000/ingest/${listType}/${listItemId}`;
```

Comment:
is this intentional?  I think localhost will not work when deployed.

Reply:
no, should be lists, mb